docs/enhancements/multidisk-ova.md

```diff
@@ -0,0 +1,79 @@
+---
+title: multidisk-ova
+authors:
+  - "@mansam"
+reviewers:
+  - "@mnecas"
+approvers:
+  - "@mnecas"
+creation-date: 2024-11-26
+last-updated: 2024-11-26
+status: implementable
+---
+
+# Storage Mapping for multi-disk OVA Imports
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] User-facing documentation is created
+
+## Summary
+
+In the current release of Forklift it is possible to import OVA appliances
+that have been exported from a vSphere environment.  However, all of
+an appliance's disks are mapped to the same storage class, and it is
+not possible for the end user to indicate that certain disks should be
+mapped differently from others. In order to support more use cases, it
+should be possible to select map a destination storage class for each
+disk of an OVA appliance individually. Forklift does not normally allow
+such fine-grained storage mapping because it becomes untenable with a
+large number of VMs, but because an OVA plan is expected to involve only
+a small number of VMs it should be acceptable.
+
+### Goals
+
+* Forklift will enable end users to individually map each disk of a
+vSphere OVA appliance to a destination storage class of their choice.
+
+### Non-Goals
+
+* Forklift will not allow fine-grained storage mapping for any VM providers
+other than OVA.
+
+## Proposal
+
+The OVA inventory adapter should be modified to surface each disk from each
+OVA appliance as its own source storage class. The OVA VM builder would then
+be modified to respect these mappings when building the DataVolumes rather
+than assuming only a single mapping.
+
+### Security, Risks, and Mitigations
+
+No new security risks are introduced by permitting disks to be mapped individually.
```

Comment:
As a riks I would concider backwards compatibility of the migrations plans.

Reply:
Addressed in the Upgrade/Downgrade portion of the enhancement due to the lack of a security impact.